draft-irtf-qirg-principles-09-rc.xml

```diff
@@ -1548,22 +1580,37 @@ QR - quantum repeater
            offered by quantum entanglement is one of the key driving factors.
            <vspace blankLines="1" />
 
-           It turns out that as long as the underlying implementation
-           corresponds to (or sufficiently approximates) theoretical models of
-           quantum cryptography, quantum cryptographic protocols do not need
-           the network to provide any guarantees about the confidentiality or
-           integrity of the transmitted qubits or the generated entanglement.
-           Instead, applications, such as QKD, establish such guarantees in an
-           end-to-end fashion using the classical network in conjunction with
-           the quantum one.
+           Fortunately, from an application's point of view, as long as the
+           underlying implementation corresponds to (or sufficiently
+           approximates) theoretical models of quantum cryptography, quantum
+           cryptographic protocols do not need the network to provide any
+           guarantees about the confidentiality or integrity of the transmitted
+           qubits or the generated entanglement. Instead, applications will
```

Comment:
RE> quantum cryptographic protocols do not need the network to provide any guarantees about the confidentiality or integrity of the transmitted qubits

But authentication is required (and this is non-trivial).  Specifically, the classical channel for post-processing is required to be an authenticated channel.